tasks/orka-deploy.yaml

```diff
@@ -33,6 +47,11 @@ spec:
 
         Set to false if you intend to clean up VMs after use manually.
       default: "true"
+    - name: orka-image
+      type: string
+      description: |
+        The name of the docker image which runs the task step.
+      default: docker.io/macstadium/orka-tekton-runner:2020-10-23-a93757dc-0.1@sha256:e8ed3370dcb91cdb8bcd4e9a7e9f6652879d80acdab9644cda69a98cd8c93339
```

Comment:
For master branch - we should set this to `docker.io/macstadium/orka-tekton-runner:master`. Same for everywhere else the image is referenced.

For versioned releases - we will definitely want to pin the image tag to the hash.